docs/en/ingest-management/tab-widgets/add-fleet-server/content.asciidoc

```diff
@@ -6,7 +6,7 @@ deployment.
 
 To confirm that an {integrations-server} is available in your deployment:
 
-. In {kib}, go to **Management > {fleet}**.
+. In {kib} sidebar, go to **Management > {fleet}**.
```

Comment:
A couple of thoughts:

* Our guidelines recommend that we avoid using the term "sidebar". To follow the guidelines, we should probably say something like, "In {kib}, open the main menu, then click {fleet}." (not sure we need to mention the management container)
![image](https://user-images.githubusercontent.com/14206422/215841364-fe3904a1-46bb-46a2-9aca-d0a16d2dc4c9.png)

* We should make this change globally rather than in just one topic.
* There's already [an issue open to make the tasks more consistent](https://github.com/elastic/observability-docs/issues/2278). You could close this PR and add a comment to that issue instead so that we fix all occurrences of the problem.